### PR TITLE
Update JavaEnclave's license declaration

### PR DESCRIPTION
**Pull request by superajun-wsj**

### CODE REVIEW

**Review by ziyilin — APPROVED** [review]

**Review by yuleil — APPROVED** [review]

File: licenses/LICENSE-graal.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+          Product License  - GraalVM Community Edition
+
+This is a release of GraalVM Community Edition. GraalVM Community Edition
+consists of multiple modules. The software as a whole, as well as the JVMCI
+and VisualVM modules, are released under version 2 of the GNU General Public
+License with the “Classpath” Exception.
+The text of the foregoing licenses is reproduced below.
+
+Copyright (c) 2015, 2019, Oracle and/or its affiliates. All rights reserved.
```
Comment (mssun) [inline licenses/LICENSE-graal.txt:9]: This is not the full text of the Graal license. Please copy all (https://github.com/oracle/graal/blob/master/LICENSE) here.

Comment (superajun-wsj) [reply licenses/LICENSE-graal.txt:9]: Done

**Comment by mssun** [comment]
Can you create a new package and vote against it?

**Comment by superajun-wsj** [comment]
> Can you create a new package and vote against it?

Hi mssun, OK. I will vote again, thanks very much for your comments.

**Pull request merged**